### PR TITLE
run flutter update-packages --force to Update all packages

### LINKED ISSUE #85035: Breakpoints are ignored when using custom base url
I'm having a trouble with breakpoints when using flutter for web in debug mode. Everything worked fine until I had to change **base** url attribute because our application is not sitting root. 

Of course that this could be fixed by changing base during development but it is really cumbersome.  For my use case I have written a custom CI stage that changes base url before build without committing so its not a blocker.

I haven't found similar issue so I even tho it is not a blocker I decided to post an issue to possibly save some time for others when they encounter this issue. 

## Steps to Reproduce

1. Create new flutter application using `flutter create bug`.
2. Change base from `<base href="/">` to `<base href="/bug/">` inside of `<app root>/web/index.html`
3. Place a breakpoints somewhere in your application using your favourite IDE (im using **intellij** but it also happens in **vs code**)
4. Run your application in debug
5. Watch how your breakpoint is ignored

**Expected results:** My breakpoint wouldn't be ignored

**Actual results:** My breakpoints are sadly ignored

```
❯ flutter doctor -v
[✓] Flutter (Channel stable, 2.2.2, on macOS 11.4 20F71 darwin-x64, locale en-GB)
    • Flutter version 2.2.2 at /Users/_/Development/flutter
    • Framework revision d79295af24 (11 days ago), 2021-06-11 08:56:01 -0700
    • Engine revision 91c9fc8fe0
    • Dart version 2.13.3

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/_/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.5.1, Build version 12E507
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] IntelliJ IDEA Ultimate Edition (version 2021.1.1)
    • IntelliJ at /Applications/IntelliJ IDEA.app
    • Flutter plugin version 57.0.5
    • Dart plugin version 211.7233

[✓] VS Code (version 1.57.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.23.0

[✓] Connected device (1 available)
    • Chrome (web) • chrome • web-javascript • Google Chrome 91.0.4472.114

• No issues found!
```

### LINKED ISSUE #102159: Flutter tools debugging with web server device is not working with chrome 100.x
See comment in https://github.com/flutter/flutter/issues/101224#issuecomment-1097565580

The issues are caused by https://github.com/dart-lang/webdev/issues/1581

Related: https://github.com/flutter/flutter/issues/101224

/cc @fhuonder

### PR DESCRIPTION
**Pull request by annagrin**
Update all packages (to bring in dwds14.0.0)

- pass base path to the `FrontendServerStrategyProvider`

Closes: https://github.com/flutter/flutter/issues/102159
Closes: https://github.com/flutter/flutter/issues/85035

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: dev/integration_tests/flutter_gallery/lib/gallery/about.dart [hunk 1]
```diff
@@ -25,7 +25,7 @@ class _LinkTextSpan extends TextSpan {
   _LinkTextSpan({ super.style, String? url, String? text }) : super(
     text: text ?? url,
     recognizer: TapGestureRecognizer()..onTap = () {
-      launch(url!, forceSafariVC: false);
+      launchUrlString(url!);
```
Comment (annagrin) [inline dev/integration_tests/flutter_gallery/lib/gallery/about.dart:28]: @stuartmorgan I updated a few deprecated APIs in this directory, could you please have a look - I am not sure if parameters match correctly.

**Comment by christopherfujino** [comment]
Thanks for opening this! However, I've already fixed most of ht failing tests on https://github.com/flutter/flutter/pull/103220, so I'm going to close this PR and patch the changes into mine.

**Comment by annagrin** [comment]
@christopherfujino sounds good, thanks!

**Pull request closed without merging**